lxd/daemon.go

```diff
@@ -1158,6 +1158,33 @@ func (d *Daemon) init() error {
 	d.startStopLock.Lock()
 	defer d.startStopLock.Unlock()
 
+	// Setup and load the server's UUID file.
```

Comment:
Not a blocker, but this would have been a great opportunity to reduce cyclomatic complexity by moving this to a separate function on `Daemon` that can be called from here.